Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/08-DNS-Records/main.tf

```diff
@@ -0,0 +1,27 @@
+########
```

Comment:
Please indicate at the top of this file that this is only used for the B/G scenario

Reply:
fixed